ecosystem/sep-0010.md

```diff
@@ -97,12 +99,20 @@ On success the endpoint must return `200 OK` HTTP status code and a JSON object
   * source account set to server's signing account
   * invalid sequence number (set to 0) so the transaction cannot be run on the Stellar network
   * time bounds: `{min: now(), max: now() + 900 }` (we recommend expiration of 15 minutes to give user time to sign transaction)
-  * operations: `manage_data(source: client_account, key: '<home domain> auth', value: random_nonce())`
-    * The value of key is the fully qualified domain name of the web service requiring authentication followed by `auth`. It can be at most 64 characters.
-    * The value must be 64 bytes long. It contains a 48 byte cryptographic-quality random string encoded using base64 (for a total of 64 bytes after encoding).
+  * operations:
+    * `manage_data(source: client_account, key: '<home domain> auth', value: random_nonce())`
+      * The value of key is the fully qualified domain name of the web service requiring authentication followed by `auth`. It can be at most 64 characters.
+      * The value must be 64 bytes long. It contains a 48 byte cryptographic-quality random string encoded using base64 (for a total of 64 bytes after encoding).
+    * `manage_data(source: server_account, key: 'server_hostname', value: server_hostname)`
+      * The value is the fully qualified domain name of the SEP-10 server that issued the challenge transaction. It can be at most 64 characters.
+    * zero or more `manage_data(source: server_account, ...)` reserved for future use
   * signature by the service's stellar.toml `SIGNING_KEY`
 * `network_passphrase`: (optional but recommended) Stellar network passphrase used by the server. This allows the client to verify that it's using the correct passphrase when signing and is useful for identifying when a client or server have been configured incorrectly.
 
+Notes:
```

Comment:
I can see the argument that the 2nd manage data op listed above is grouped with the `other` ops that are unordered. But most people will look at the `transaction` challenge description and see the explicit 2nd op and be confused.

Reply:
I've removed these notes, let's continue to discuss the issue here: https://github.com/stellar/stellar-protocol/pull/741#discussion_r502078391.